modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -311,16 +319,29 @@ object ElasticRequest {
       with HasRouting[SearchRequest]
       with WithSort[SearchRequest] {
     def aggregate(aggregation: ElasticAggregation): SearchAndAggregateRequest
+    def highlights(value: Highlights): Search
```

Comment:
I'm thinking about `highlighted(highlights: Highlights)`. What do you think? Also, add a new line before.

Reply:
Maybe to use `highlight` to have meaning as a verb. (I want this search and highlight this)

Reply:
Can you add a new line before?